### PR TITLE
Add MTE-2908 Focus Full Functional Test plan on Github Actions

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-2908)

## :bulb: Description
Let's put the full functional tests to run on Github Actions regularly.

All Full Functional Tests are passing. Test report: https://github.com/mozilla-mobile/firefox-ios/actions/runs/9849722935/job/27193929577

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: .github/workflows/focus-ios-tests.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name: "Focus Tests"
     
 on:
-    workflow_dispatch:
+  schedule:
+    - cron: "0 1 * * *"
```
Comment (clarmso) [inline .github/workflows/focus-ios-tests.yml:5]: I think we are ready to let the job running automatically.

File: .github/workflows/focus-ios-tests.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: "Focus Tests"
-    
 on:
-    workflow_dispatch:
+  schedule:
+    - cron: "0 1 * * *"
```
Comment (clarmso) [inline .github/workflows/focus-ios-tests.yml:4]: Let's run the tests everyday

File: .github/workflows/focus-ios-tests.yml [hunk 3]
```diff
@@ -95,9 +95,9 @@ jobs:
             - name: Print test report
               id: test-report
               run: |
-                python ../test-fixtures/ci/convert_junit_to_markdown.py --github --${{ env.browser }} ./build/reports/junit.xml ./github.md
+                python ../test-fixtures/ci/convert_junit_to_markdown.py ${{ matrix.xcodebuild_test_plan == 'SmokeTest' && '--smoke' || '' }} --github --${{ env.browser }} ./build/reports/junit.xml ./github.md
```
Comment (clarmso) [inline .github/workflows/focus-ios-tests.yml:98]: `--smoke` argument is to count the flaky tests. Not applicable for full functional tests.

**Review by isabelrios — APPROVED** [review]
Once conflicts are fixed. Thanks!

**Comment by clarmso** [comment]
Oops. Mess up rebase again. Will open a new branch.

**Pull request closed without merging**